src/main/kotlin/no/nav/klage/oppgave/domain/kodeverk/Tema.kt

```diff
@@ -85,6 +85,18 @@ enum class Tema(override val id: String, override val navn: String, override val
     }
 }
 
+object TemaTilgjengeligeForEktefelle {
+    private val ektefelleTemaerIProdGcp = EnumSet.of(Tema.OMS)
+    private val ektefelleTemaerIDevGcp = EnumSet.of(Tema.OMS)
+
+    fun temaerTilgjengeligForEktefelle(environment: Environment): EnumSet<Tema> =
```

Comment:
Lurer på om det er litt mye å sende inn hele `environment` her om det bare er `activeProfile` som er relevant.

Reply:
Vi gjør dette flere steder allerede. Det er mulig det kunne vært annerledes, men jeg har fått noen feil tidligere når jeg prøvde å jobbe direkte med profile, så dette er et resultat av det. 
Jeg foreslår at noen lager en egen PR på å endre fra environment til profile isåfall, for det vil gjelde flere steder i koden som sagt, ikke bare i denne PRen.